doc/api/deprecations.md

```diff
@@ -94,14 +94,6 @@ Within the [`child_process`][] module's `spawn()`, `fork()`, and `exec()`
 methods, the `options.customFds` option is deprecated. The `options.stdio`
 option should be used instead.
 
-<a id="DEP0007"></a>
-### DEP0007: cluster worker.suicide
```

Comment:
-1 on using roundabout wording. We can't remove problematic terminology from _everywhere_ (for example, git history). `deprecations.md` is a file that has a very specific purpose, and most people will never look at it.

Example: someone is trying to resurrect some old code that no longer works due to a call to a non-existent method. Searching for it in `deprecations.md` would probably be the first step. If the section about a removed API doesn't _name_ the API, then there's no point in having it at all.